source/common/upstream/maglev_lb.cc

```diff
@@ -109,7 +109,12 @@ MaglevLoadBalancer::MaglevLoadBalancer(
       use_hostname_for_hashing_(
           common_config.has_consistent_hashing_lb_config()
               ? common_config.consistent_hashing_lb_config().use_hostname_for_hashing()
-              : false) {}
+              : false),
+      hash_balance_factor_(
+          common_config.has_consistent_hashing_lb_config()
+              ? PROTOBUF_GET_WRAPPED_OR_DEFAULT(common_config.consistent_hashing_lb_config(),
```

Comment:
Do you mean we just use ```PROTOBUF_GET_WRAPPED_OR_DEFAULT(common_config.consistent_hashing_lb_config(),                                           hash_balance_factor, 0)```? How does it work if ```common_config``` does not have ```consistent_hashing_lb_config```?

Reply:
if `consistent_hashing_lb_config` is not set, then the default value will be returned. This default value would *not* have `hash_balance_factor` set so I think it would do the right thing.